tests/backup/backup_delete_test.go

```diff
@@ -266,12 +266,12 @@ var _ = Describe("{IssueDeleteOfIncrementalBackupsAndRestore}", func() {
 	})
 
 	JustAfterEach(func() {
-		defer EndPxBackupTorpedoTest(contexts)
+		defer EndPxBackupTorpedoTest(scheduledAppContexts)
 		log.InfoD("Deleting the deployed apps after the testcase")
 		// Cleaning up applications created
 		opts := make(map[string]bool)
 		opts[SkipClusterScopedObjects] = true
-		ValidateAndDestroy(contexts, opts)
+		ValidateAndDestroy(scheduledAppContexts, opts)
```

Comment:
Why do you need ValidateAndDestroy() instead use `DestoryApps()` as it's already in cleanup no point of validating?